### DISCUSSION TITLE
Forecast Data Points in a multi-series chart - only for single series

### DISCUSSION THREAD

**Discussion by UclNikolaj** [apexcharts/apexcharts.js#3519] [Q&A]
I’m trying to use the ForecastDatapoints option, but only for a specific series in my chart, but as far as I can tell, it’s not possible..?

I have this chart, and I only want the forecastdatapoints applied to a single series.

![billede](https://user-images.githubusercontent.com/99953944/204249431-31752c92-e0e8-4ed8-9247-d578cf251542.png)

any help would be appreciated

**Comment by vaishnavigitsavvy** [comment]
![image](https://github.com/apexcharts/apexcharts.js/assets/146183872/b462325f-32e4-4474-bc8d-bc33b48e9376)
I am also facing the same issue I want forecastDatapoints to be applied to bar chart and not line chart(in yellow color)- for mixed charts
